README.md

```
@@ -1,24 +1,39 @@
-<p align="center"><img src="https://dydx.exchange/icon.svg?" width="256" /></p>
+<p align="center"><img src="https://raw.githubusercontent.com/wt-xyz/starboard/8b3ee6a20850848ec4fb441a9e7b820b2cafc0b0/public/starboard-symbol-yellow.svg" width="512" /></p>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)